content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -219,57 +230,63 @@ To configure the Tracer using a JSON file, create `datadog.json` in the instrume
+### Unified Service Tagging
 
-| Setting Name                                        | Description                                                                                                                                                                                                       |
+To use [Unified Service Tagging][7], configure the `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` settings for your services. 
+
+| Setting Name                                        |Description                                                                                                                                                                                                       |
 |-----------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `DD_ENV`<br/><br/>`Environment`                     | If specified, adds the `env` tag with the specified value to all generated spans. See [Agent configuration][8] for more details about the `env` tag. Available for versions 1.17.0+                                                           |
-| `DD_SERVICE`<br/><br/>`ServiceName`            | If specified, sets the service name. Otherwise, the .NET Tracer tries to determine service name automatically from application name (e.g. IIS application name, process entry assembly, or process name). Available for versions 1.17.0+  |
-| `DD_VERSION`<br/><br/>`ServiceVersion`            | If specified, sets the version of the service. Available for versions 1.17.0+
-| `DD_TAGS`<br/><br/>`GlobalTags`       | If specified, adds all of the specified tags to all generated spans (e.g., `layer:api,team:intake`). Available for versions 1.17.0+                                                                                                                                              |
+| `DD_ENV`<br/><br/>`Environment`                     | If specified, adds the `env` tag with the specified value to all generated spans. Available for versions 1.17.0+                                           |
+| `DD_SERVICE`<br/><br/>`ServiceName`                 | If specified, sets the service name. Otherwise, the .NET Tracer tries to determine service name automatically from application name (e.g. IIS application name, process entry assembly, or process name). Available for versions 1.17.0+      |
+| `DD_VERSION`<br/><br/>`ServiceVersion`              | If specified, sets the version of the service. Available for versions 1.17.0+
+| `DD_TAGS`<br/><br/>`GlobalTags`                     | If specified, adds all of the specified tags to all generated spans (e.g., `layer:api,team:intake`). Available for versions 1.17.0+                                                                                                                                              |
```

Comment:
Only `env`, `service`, and `version` are specific to "Unified Service Tagging". `DD_TAGS` should probably be in the table with other tracer settings.
See https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging